css/support/numeric-testcommon.js

```diff
@@ -122,6 +130,10 @@ function test_math_specified(testString, expectedString, {approx, msg, msgExtra,
         assert_not_equals(expectedValue, '', `${expectedString} isn't valid in '${prop}'; got the default value instead.`)
         if (approx) {
             let extractValue = function(value) {
+                let escapeRegex = (string) => {
+                    return string.replace(/[/\-\\^$*+?.()|[\]{}]/g, "\\$&");
```

Comment:
This probably needs a comment... But is this correct? The previous code would deal with the used `transform` values (so, `matrix(...)`, while this code seems like it wouldn't).